it/common/src/main/java/com/google/cloud/teleport/it/common/AbstractPipelineLauncher.java

```diff
@@ -198,8 +212,12 @@ protected JobState handleJobState(Job job) {
   }
 
   /** Creates a JobInfo object from the provided parameters. */
-  protected LaunchInfo getJobInfo(LaunchConfig options, JobState state, Job job, String runner) {
+  protected LaunchInfo getJobInfo(LaunchConfig options, JobState state, Job job) {
     Map<String, String> labels = job.getLabels();
+    String runner =
+        job.getEnvironment().getExperiments().contains("use_runner_v2")
```

Comment:
is it null safe if there are no experiments?

Reply:
Ah, good catch!

Changed the code to make it nullsafe